Spring-Boot/Spring-Boot-Student/src/main/java/tw/waterball/judgegirl/springboot/student/repositories/jpa/JpaStudentDataPort.java

```diff
@@ -30,4 +32,9 @@
     Optional<StudentData> findStudentById(Integer id);
 
     boolean existsByEmail(String email);
+
+    Page<StudentData> findByIsAdminFalse(Pageable pageable);
+
+    Page<StudentData> findByIsAdminTrue(Pageable pageable);
```

Comment:
merge them into one so you won't need if-else

`(boolean isAdmin, Pageable pageable)`

Reply:
Rename all the `isAdmin` to `admin` 

We seldom name our boolean attribute starting with `is`. This was a mistake.
Only boolean's getters will start its name with `is`